WIP: Zowe HA documentation changes (#1688)
zowe/docs-site: docs/getting-started/overview.md

* Consistent Access: API routing and standardization of API service URLs through the Gateway component provides users with a consistent way to access mainframe APIs at a predefined address.
* Dynamic Discovery: The Discovery Service automatically determines the location and status of API services.
* High-Availability: API Mediation Layer is designed with high-availability of services and scalability in mind.

> * Caching Service: This feature is designed for Zowe components in a high availability configuration. It supports the High Availability of all components within Zowe, components either need to be stateless, or offload the state to a location accessible by all instances of the service, including those which just started.

Reviewer: Perhaps this instead:
```suggestion
* Caching Service: This feature is designed for Zowe components in a high availability configuration. It supports the High Availability of all components within Zowe, **allowing components to be stateless by providing a mechanism to offload their** state to a location accessible by all instances of the service, including those which just started.
```
